src/server/common/model/answer/hidden/hidden.js

```diff
@@ -0,0 +1,91 @@
+import { AnswerModel } from '../answer-model.js'
+import { NotImplementedError } from '../../../helpers/not-implemented-error.js'
+
+/** @import {AnswerViewModelOptions} from '../answer-model.js' */
+
+/**
+ * export @typedef {{
+ *  payloadKey: string,
+ *  value: string,
+ * }} HiddenAnswerConfig
+ */
+
+/**
+ * @typedef {string} TextData
+ */
+
+/**
+ * @template Payload
+ * @augments {AnswerModel<Payload>}
+ */
+export class HiddenAnswer extends AnswerModel {
+  // eslint-disable-next-line jsdoc/require-returns-check
+  /** @returns {HiddenAnswerConfig} */
+  get config() {
+    return /** @type {any} */ (this.constructor).config
+  }
+
+  // eslint-disable-next-line jsdoc/require-returns-check
+  /** @returns {HiddenAnswerConfig} */
+  static get config() {
+    throw new NotImplementedError()
+  }
+
+  /**
+   * @returns {string | undefined}
+   */
+  get value() {
+    return this._data?.[this.config.payloadKey]
+  }
+
+  // eslint-disable-next-line @typescript-eslint/class-literal-property-style
+  get template() {
+    return 'model/answer/hidden/hidden.njk'
+  }
+
+  /**
+   * @returns {TextData}
+   */
+  toState() {
+    return this.value ?? ''
+  }
+
+  validate() {
+    return { isValid: !!this.value, errors: {} }
```

Comment:
`this.value === this.config.value`?  I think technically they're the same thing - so honestly this is probably just that this feels a bit indirect to me.

Reply:
This is just checking that the value is set to something (it is truthy)

Reply:
But might be safer to check it is the same as the configured value. I'll update it.